exporters/elasticsearch/CMakeLists.txt

```diff
@@ -3,6 +3,9 @@ include_directories(${CMAKE_SOURCE_DIR}/ext/include)
 
 add_library(opentelemetry_exporter_elasticsearch_logs src/es_log_exporter.cc)
 
+target_link_libraries(opentelemetry_exporter_elasticsearch_logs
```

Comment:
Seems this change is only for MinGW, is it needed for other toolchain?

Reply:
It builds with msvc, so I assume it's unique to MinGW